docs/3.0.0-release-note.md

```diff
@@ -250,7 +250,7 @@ ce930e7a6 [Jaxrs-cxf] Add bean-level cascaded beanvalidation for pojos (@Valid)
 ff1178ad7 [Java][Spring] fix missing optional query params
 2103fadab Fix package declaration for play-framework
 2c6380c84 fix inner item (list, map) for play framework
-e33b350c8 Fix an issue in Play Framework generator where a CSV is empty and transfered to the controllerImp with an empty item.
+e33b350c8 Fix an issue in Play Framework generator where a CSV is empty and transferred to the controllerImp with an empty item.
```

Comment:
@scop thanks. I've updated the release note in https://github.com/OpenAPITools/openapi-generator/releases/tag/v3.0.0 accordingly